src/Adyen/Util/OpenInvoice.php

```diff
@@ -0,0 +1,84 @@
+<?php
+namespace Adyen\Util;
+
+class OpenInvoice
+{
+    const AFTERPAY_PAYMENT_METHOD = 'afterpay';
+    const AFTERPAY_B2B_PAYMENT_METHOD = 'afterpay_b2b';
+    const AFTERPAY_DEFAULT_PAYMENT_METHOD = 'afterpay_default';
+    const AFTERPAY_DIRECTDEBIT_PAYMENT_METHOD = 'afterpay_directdebit';
+
+    const AFTERPAYTOUCH_PAYMENT_METHOD = 'afterpaytouch';
+
+    const RATEPAY_PAYMENT_METHOD = 'ratepay';
+    const RATEPAY_DIRECTDEBIT_PAYMENT_METHOD = 'ratepay_directdebit';
+
+    const KLARNA_PAYMENT_METHOD = 'klarna';
+    const KLARNA_B2B_PAYMENT_METHOD = 'klarna_b2b';
+    const KLARNA_PAYNOW_PAYMENT_METHOD = 'klarna_paynow';
+    const KLARNA_ACCOUNT_PAYMENT_METHOD = 'klarna_account';
+
+    /**
+     * List of open invoice payment methods
+     *
+     * @var array
+     */
+    private static $openInvoicePaymentMethods = array(
+        self::AFTERPAY_PAYMENT_METHOD,
+        self::AFTERPAY_B2B_PAYMENT_METHOD,
+        self::AFTERPAY_DEFAULT_PAYMENT_METHOD,
+        self::AFTERPAY_DIRECTDEBIT_PAYMENT_METHOD,
+        self::AFTERPAYTOUCH_PAYMENT_METHOD,
+        self::RATEPAY_PAYMENT_METHOD,
+        self::RATEPAY_DIRECTDEBIT_PAYMENT_METHOD,
+        self::KLARNA_PAYMENT_METHOD,
+        self::KLARNA_B2B_PAYMENT_METHOD,
+        self::KLARNA_PAYNOW_PAYMENT_METHOD,
+        self::KLARNA_ACCOUNT_PAYMENT_METHOD,
+    );
+
+    /**
+     * For Klarna And AfterPay use Vat category High others use none
+     *
+     * @param $paymentMethod
+     * @return string 'High'/'None'
+     */
+    public function getVatCategory($paymentMethod)
+    {
+        if ($paymentMethod == "klarna" ||
+            strlen($paymentMethod) >= 9 &&
+            mb_substr($paymentMethod, 0, 9) == 'afterpay_'
```

Comment:
I'm not really sure how it was working before but in this scenario only klarna payment and afterpay_b2b, afterpay_deafult and afterpay_directdebit is taken into consideration so klarna_* and afterpay is not. So I propose the following:


```suggestion
        if (mb_substr($paymentMethod, 0, 6) == self::KLARNA_PAYMENT_METHOD ||
            mb_substr($paymentMethod, 0, 8) == self::AFTERPAY_PAYMENT_METHOD
```